src/content/workshops/functions/1-what-is-serverless/_index.md

```diff
@@ -4,12 +4,105 @@ weight: 101
 outputs: ["Reveal"]
 ---
 
-# Slide 1
+## What is Serverless?
 
-Test
+- Execution model
+- Pay as you consume
+- No need to control scaling
+
+{{% note %}}
+Serverless is an execution model. Everytime you hear the word serverless, it's talking about this cloud focused execution model. Running the same code locally is a different concept.
+
+An attribute of that execution model is that you pay for what you consume and not for what you reserve. When you create services in cloud providers, you reserve for a certain amount of CPU/RAM. This is not a necessary concern when you start a new serverless application.
+
+Serverless goes without this as the whole premise to care about servers less. Meaning that when using a serverless framework, you should not be required to control scaling at all.
+
+Now, how do you run code on a Serverless platform? This takes us to...
+
+{{% /note %}}
+
+---
+
+## Serverless Runtime
+
+Available Runtimes:
+
+- Azure Functions
+- AWS Lambda
+- Google Cloud Functions
+- IBM Cloud Functions
+- ...
```

Comment:
OpenFaas could be interesting to mention as well.

Reply:
Good catch. Might as well add an open-source project to this as well.